storage/grpc_client.go

```diff
@@ -591,3 +586,10 @@ func (c *grpcStorageClient) CreateHMACKey(ctx context.Context, desc *hmacKeyDesc
 func (c *grpcStorageClient) DeleteHMACKey(ctx context.Context, desc *hmacKeyDesc, opts ...storageOption) error {
 	return errMethodNotSupported
 }
+
+// setUserProjectMetadata appends a project ID to the outgoing Context metadata
```

Comment:
Worth noting that this is for billing purposes, and generally optional except for requester-pays buckets.

Reply:
Update! Thanks.